### ISSUE TITLE
Alter table with connection to read-only node successfull

### ISSUE THREAD

**Issue by DLT1412** [crate/crate#7068]
**CrateDB version**:
2.3.2

Please look at image attached.
<img width="608" alt="screen shot 2018-03-22 at 7 10 06 pm" src="https://user-images.githubusercontent.com/8294986/37769811-d61a75d2-2e04-11e8-983a-a5840b90b3d9.png">

**Comment by m-kharbat** [comment]
Thank you for reporting this issue, we will look into it.
In the meanwhile, please provide us with the following missing information

```
 - JVM version: java -version 
 - Kernel: uname -a
 - Distribution: cat /etc/os-release
 - Docker: docker \-\-version
 - Number of nodes; Please mention special master/data/client node configuration
```

**Comment by DLT1412** [comment]
JVM version:  openjdk version 1.8.0_111
Kernel: 3.10.0-327.36.3.el7.x86_64
OS: Centos 7
Conf:

> node.master:  false
> node.data: false

**Comment by chaudum** [comment]
Hi @DLT1412 
The bug has been fixed and merged both into master and 2.3. Will be available with the next 2.3.x release. Thanks again for reporting.
